tests/sentry/snuba/metrics/test_naming_layer.py

```diff
@@ -60,20 +60,8 @@ def test_invalid_public_name_regex(name):
             ParsedMRI("s", "sessions", "error", "none"),
         ),
         (
-            "dist:my_namespace/organizations/v1/my endpoint@{none}",
-            ParsedMRI("dist", "my_namespace", "organizations/v1/my endpoint", "{none}"),
-        ),
-        (
-            "d:transactions/measurements.disk_io@byte/second",
-            ParsedMRI("d", "transactions", "measurements.disk_io", "byte/second"),
-        ),
-        (
-            "c:custom/http.client.open_connections@{connection}",
-            ParsedMRI("c", "custom", "http.client.open_connections", "{connection}"),
-        ),
-        (
-            "c:custom/http.client.active_requests@{request}",
-            ParsedMRI("c", "custom", "http.client.active_requests", "{request}"),
+            "dist:my_namespace/organizations/v1/my endpoint@none",
+            ParsedMRI("dist", "my_namespace", "organizations/v1/my endpoint", "none"),
         ),
```

Comment:
these tests don't have valid `unit`s -- I "fixed" the `{none}` one and deleted the rest ?